verma2021explaining/openreview/content.tex

```diff
@@ -0,0 +1,294 @@
+% $t_{0 \to i} = e^{\gamma_i} \odot x + \delta_i$
+% The gammas are truncated just like the deltas and their $L1$ norm is added to the regularization term. Note that these transformations are strictly more expressive. If $\gamma$ is the zero vector, these transformations reduce to regular TGT. 
+
+In figure \ref{fig:pt_heart_housing}, we can see the difference in performance on two dataset included in the original experiments. Scaling does not seem to improve performance on the UCI Boston Housing dataset and slightly decreases performance on the UCI Heart Disease dataset. The similarity of the gammas ($\gamma_s$) is mostly above $0.9$. 
+
+In figure \ref{fig:pt_diabetes_breast},  we see the same metrics for the Breast Cancer and Pima Indians Diabetes Dataset. For the Diabetes dataset, the performance improves slightly and the gammas($\gamma_s$) show high similarity. For the Breast Cancer dataset, the performance is about the same but the gammas($\gamma_s$) show relatively low similarity. 
+
+Altogether, these results suggest that the addition of scaling does not significantly improve the accuracy and correctness while making the transformations more complex. Based on our experiments, we do not recommend the addition of scaling in the explanation functions, and conclude that the original TGT is expressive enough. 
+
+\subsubsection{Experiment with Modified Synthetic Data}
+In order to study the efficacy of the proposed scaling function, we perform experiments on the synthetic dataset. We modify one of the groups of points $G$ by performing the operation $ax_{i}^{k} + b$, where $i$ corresponds to the group number and $k$ denotes the feature dimension which we modify. We define $a \sim \mathcal{U}(1.0, 2.0)$  and $b \sim \mathcal{U}(-0.5, 1.0)$. We add modified group $G'$ into the original data $D$ to get the new data $D^{'}$. We follow the experiment setup from the original paper as: a) $r(G^{'})$ should form a different group of its own. b) $G^{'}$ should be within the distribution of the original $D$. In this study, we want to investigate whether the TGT with scaling is able to recover the modifications, and if in doing so it affects the explanations between other groups. The sampling procedure gave a=2.0, b=0.60 and we keep k=0. We observe that the explanations with scaling are able to recover the modification to an approximate degree(scaling factor $e^{\gamma} \approx 2.38$, actual a=2.0), and give better correctness as compared to the regular TGT (refer figure \ref{tab:synth_modified_results}). Interestingly, the translations explanations of the scaling extension are approximately equal to the deltas of the regular TGT. The exact results can be found in Table \ref{tab:synth_modified_results}. Figures \res \ref{fig:synth_modified} and \ref{fig:synth_modified_change} in the appendix show the data spread and resulting translations. 
+
```

Comment:
Latex compilation error at: "Figures \res \ref{fig:synth_modified} and ..." -- `\res` is undefined, please remove it.